darwin/gal/Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "gal",
+    platforms: [
+        .iOS("11.0"),
+        .macOS("11.0")
+    ],
+    products: [
+        .library(name: "gal", targets: ["gal"])
+    ],
+    dependencies: [],
+    targets: [
+        .target(
+            name: "gal",
+            dependencies: [],
+            resources: [
+                .process("PrivacyInfo.xcprivacy"),
+            ]
```

Comment:
This was not added previously, I'm unsure which APIs require this though it seems most Flutter plugins have this including [url_launcher](https://github.com/flutter/packages/blob/main/packages/url_launcher/url_launcher_ios/ios/url_launcher_ios/Sources/url_launcher_ios/Resources/PrivacyInfo.xcprivacy). This will become required even for macOS (see https://github.com/flutter/packages/pull/7687).

It's likely that saving images to the Gallery and requesting permission need this but need to double-check.